### DISCUSSION TITLE
Equation missing in documentation

### DISCUSSION THREAD

**Discussion by prvmalik** [Open-Systems-Pharmacology/Forum#955] [Questions & Problems]
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-defining-inhibition-induction-processes

Section on Induction

**Comment by StephanSchaller** [comment]
Hi Paul,

You can create issues for the documentation when you click on "Edit on GitHub" in the GitHub Docs:
![image](https://user-images.githubusercontent.com/25477643/129973880-91f1f996-9277-4faa-9a47-c7d3a186e76a.png)


Ideally, you can directly edit the section on GitHub and create a pull request for the changes. ;-)
![image](https://user-images.githubusercontent.com/25477643/129973802-61d96b82-3a22-441e-bd63-834feddca6d1.png)

Cheers,
Stephan

**Comment by prvmalik** [comment]
I thought someone may have a quicker answer than me figuring it out. I will figure out next week and update.
